index.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
         "indent": ["error", 4, { "SwitchCase": 1 }],
         "jsx-quotes": ["error", "prefer-double"],
         "padded-blocks": ["off", "never"],
-        "id-length": ["error", {"exceptions": ["b", "_"]}],
+        "id-length": ["error", {"exceptions": ["_", "x", "y", "z"]}],
```

Comment:
removing `b` will lead to hundrets of new errors in insights due to this `bem` thingy .. is it really worth it?

Reply:
yes, i think so, because we want to remove bem. lets add 'rename b to bem' to the tasklist of the insights issue here https://github.com/pmedianetwork/adverity/issues/9342

Reply:
or remove bem